middlewares/error.js

```diff
@@ -2,9 +2,9 @@ const defaultErrorStatus = 500;
 
 const errorMiddleware = (err, req, res, next) => {
```

Comment:
Нужно исправить: ошибки линтинга.
```bash
% npx eslint --ext .js .

/Users/jmi/work/ya-reviews/backend-mesto/middlewares/error.js
  3:41  error  'next' is defined but never used  no-unused-vars

✖ 1 problem (1 error, 0 warnings)
```
Решается добавлением строки в список игнорирования:
```suggestion
// eslint-disable-next-line no-unused-vars
const errorMiddleware = (err, req, res, next) => {
```

Reply:
Исправил.